maintenance/openshift-upgrade.md

```diff
@@ -31,3 +31,8 @@ You can now monitor the upgrade progress with the following command:
 ```bash
 watch oc get tigerastatus
 ```
+
+If you were upgrading from a version of Calico prior to v3.14 and followed the pre-upgrade steps for host endpoints above, review traffic logs from the temporary policy,
+add any global network policies needed to whitelist traffic, and delete the temporary network policy **allow-all-upgrade*.
+
+{% include content/auto-hostendpoints-migrate.md orch="OpenShift" %}
```

Comment:
Is there any sane way to have the Openshift doc use `oc` instead of `kubectl`?

Reply:
c3dd0d7 and 0c919e5